cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -106,171 +103,197 @@ New Features
+
+- `CDAP-4548 <https://issues.cask.co/browse/CDAP-4548>`__ -
+  Added a config setting 'master.collect.app.containers.log' that can be set to 'false' to
+  disable streaming of application logs back to the CDAP Master log.
 
+- `CDAP-4598 <https://issues.cask.co/browse/CDAP-4598>`__ -
+  Added an error message when a required field is not provided when configuring Hydrator
+  pipeline.
+  
 Bug Fixes
 ---------
-- `CDAP-3694 <https://issues.cask.co/browse/CDAP-3694>`__ -
-  Fixed test failures in the PurchaseHistory, StreamConversion, and WikipediaPipeline example apps included in the sdk.
+- `CDAP-1174 <https://issues.cask.co/browse/CDAP-1174>`__ -
+  Prefix start script functions with ``'cdap'`` to prevent namespace collisions.
 
-- `CDAP-4240 <https://issues.cask.co/browse/CDAP-4240>`__ -
-  Fixed a bug in the CLI command to load an artifact, where the wrong artifact name and version is
-  used if the artifact name ends with a number.
+- `CDAP-2470 <https://issues.cask.co/browse/CDAP-2470>`__ -
+  Added a check to cause a DB (source or sink) pipeline to fail during deployment if the
+  table (source or sink) was not found, or if an incorrect connection string was provided.
 
-- `CDAP-4294 <https://issues.cask.co/browse/CDAP-4294>`__ -
-  Fixed a bug where plugins from another namespace were visible when creating an application using a system artifact.
+- `CDAP-3345 <https://issues.cask.co/browse/CDAP-3345>`__ -
+  Fixed a bug where the TTL for datasets was incorrect; it was reduced by (a factor of
+  1000) after an upgrade. After running the upgrade tool, please make sure the TTL values
+  of tables are as expected.
 
-- `CDAP-4092 <https://issues.cask.co/browse/CDAP-4092>`__ -
-  Fixes an issue that causes worker threads to go into an infinite recursion while exceptions
-  are being thrown in channel handlers.
+- `CDAP-3542 <https://issues.cask.co/browse/CDAP-3542>`__ -
+  Fixed an issue where the failure of a program running in a workflow fork node was
+  causing other programs in the same fork node to remain in the RUNNING state, even after
+  the Workflow was completed.
 
-- `CDAP-3924 <https://issues.cask.co/browse/CDAP-3924>`__ -
-  Fixed QueryClient to work against HTTPS.
+- `CDAP-3694 <https://issues.cask.co/browse/CDAP-3694>`__ -
+  Fixed test failures in the PurchaseHistory, StreamConversion, and WikipediaPipeline
+  example apps included in the CDAP SDK.
 
 - `CDAP-3742 <https://issues.cask.co/browse/CDAP-3742>`__ -
-  Fixed a bug where certain MapReduce metrics were not being properly emitted when using multiple outputs.
+  Fixed a bug where certain MapReduce metrics were not being properly emitted when using
+  multiple outputs.
 
-- `CDAP-4067 <https://issues.cask.co/browse/CDAP-4067>`__ -
-  Fixed an issue where socket connections to the TransactionManager were not being closed.
+- `CDAP-3761 <https://issues.cask.co/browse/CDAP-3761>`__ -
+  Fixed a problem with DBSink column names being used to filter input records fields
```

Comment:
Fixed with

"Fixed a problem with DBSink column names not being used to filter input record fields
  before writing to a DBSink."